streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1057,6 +1063,69 @@ public void shouldUpdateStandbyTask() throws Exception {
         assertEquals(0, thread.standbyRecords().size());
     }
 
+    @Test
+    public void shouldCreateStandbyTask() {
+        final MockProcessor mockProcessor = new MockProcessor();
+        internalTopologyBuilder.addSource(null, "source1", null, null, null, topic1);
+        internalTopologyBuilder.addProcessor("processor1", () -> mockProcessor, "source1");
+        internalTopologyBuilder.addStateStore(new MockKeyValueStoreBuilder("myStore", true), "processor1");
+        final StreamThread.StandbyTaskCreator standbyTaskCreator = createStandbyTaskCreator(internalTopologyBuilder);
+
+        final StandbyTask standbyTask = standbyTaskCreator.createTask(
```

Comment:
The standby task could also be reused IIUC

Reply:
Ack